text/0037-ternary-numeric-conversion.md

```diff
@@ -26,28 +26,81 @@ assert_eq!(x, y);
 This RFC introduces the following trait implementations:
 
 ```rust
-// Binary to ternary
-impl<I: AsPrimitive<i64>, T: RawEncodingBuf> From<I> for TritBuf<T> {}
+// Signed binary to ternary
+impl<T: RawEncodingBuf> From<i64> for TritBuf<T> where T::Slice: RawEncoding<Trit = Btrit> {}
+impl<T: RawEncodingBuf> From<i32> for TritBuf<T> where T::Slice: RawEncoding<Trit = Btrit> {}
+impl<T: RawEncodingBuf> From<i16> for TritBuf<T> where T::Slice: RawEncoding<Trit = Btrit> {}
+impl<T: RawEncodingBuf> From<i8> for TritBuf<T> where T::Slice: RawEncoding<Trit = Btrit> {}
 
-// Ternary to binary (these cannot be implemented as above due to Rust's orphan rules)
+// Signed ternary to binary
```

Comment:
Would it make more sense to use balanced instead of signed ?

Reply:
Will change this